lib/ansible/modules/network/ldap_entry.py

```diff
@@ -101,6 +101,12 @@
     default: present
     description:
       - The target state of the entry.
+  validate_certs:
+    required: false
+    choices: [true, false]
+    default: true
+    description:
+      - If false, will ignore self-signed certificates of server and connect
```

Comment:
It should be (decoration around the `no` and period at the end of the sentence):

```
- If set to C(no), it will ignore self-signed certificates of server and connect.
```